spec/ParseLiveQueryServer.spec.js

```diff
@@ -944,7 +1021,7 @@ describe('ParseLiveQueryServer', function () {
       expect(toSend.object).toBeDefined();
       expect(toSend.original).toBeDefined();
       done();
-    }, jasmine.ASYNC_TEST_WAIT_TIME);
+    }, ASYNC_TEST_WAIT_TIME);
```

Comment:
Why are the timeouts changed?

Reply:
There was no timeout before: the variable was undefined. As promises are all mocked with instant resolution, we can remove it completely. I only have replaced those by a small timeout to ensure that tested process were fully executed before expecting anything.

Reply:
Interesting, if it was undefined, is there maybe another issue that you have discovered? Maybe that is also why some tests are flaky.

@dplewis any ideas? You recently looked into the tests.

Reply:
I've searched for other usage of `jasmine.ASYNC_TEST_WAIT_TIME` in the project, if found none.
This is only in this file :)

Reply:
I've checked as much as I can, I only found `Promise.resolve`/`Promise.reject`, so no delay applied. But it's certainly safer to have 20-100ms of waiting time to be sure everything is completed.

I had some hard time to navigate through mocks, so I might have miss something!

Reply:
Sounds good